app/controllers/returns/ImportedPlasticPackagingController.scala

```diff
@@ -73,14 +73,12 @@ class ImportedPlasticPackagingController @Inject() (
         )
     }
 
-  private def updateAnswerAndGotoNextPage(mode: Mode, pptId: String, previousAnswers: UserAnswers, newAnswer: Boolean)
+  private def updateAnswerAndGotoNextPage(mode: Mode, pptReference: String, previousAnswers: UserAnswers, newAnswer: Boolean)
     (implicit hc: HeaderCarrier) : Future[Result] = {
-    val maybeAnswers = previousAnswers.change(ImportedPlasticPackagingPage, newAnswer)
-    val hasAnswerChanged = maybeAnswers match {
-      case None                 => Future.successful(false)
-      case Some(updatedAnswers) => cacheConnector.set(pptId, updatedAnswers).map(_ => true)
-    }
-    hasAnswerChanged.map(hasAnswerChanged => Redirect(returnsNavigator.importedPlasticPackagingRoute(mode, hasAnswerChanged, newAnswer)))
+    
+    previousAnswers
+      .change_v3(ImportedPlasticPackagingPage, newAnswer, cacheConnector.saveUserAnswerFunc(pptReference))
```

Comment:
What is change_v3?

Reply:
We added change_v2() and change_v3() as possible replacements for change() - change_v3 won